deny.toml

```diff
@@ -0,0 +1,55 @@
+# https://embarkstudios.github.io/cargo-deny/
+# cargo install cargo-deny
+# cargo update && cargo deny --all-features --log-level error check
+
+targets = [
+    { triple = "aarch64-apple-darwin" },
+    { triple = "aarch64-apple-darwin" },
+    { triple = "aarch64-linux-android" },
+    { triple = "x86_64-apple-darwin" },
+    { triple = "x86_64-pc-windows-msvc" },
+    { triple = "x86_64-unknown-linux-gnu" },
+    { triple = "x86_64-unknown-linux-musl" },
+]
```

Comment:
Is is really have to be duplicated here?

Reply:
No, not at all. I've somehow misunderstood how `targets` work I think. Fixed in  7745a23